hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -136,7 +136,7 @@
 	// constructors and factory methods ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 	protected Dialect() {
-		LOG.usingDialect( this );
+		logDeprecation();
```

Comment:
I don't think the separate method does anything here; just move that code inline.  Or at least name the method better - it does not just log deprecations

Reply:
I renamed it.